src/components/table-ajax/table-ajax.stories.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { State, Store } from '@sambego/storybook-state';
+import { enableMock } from '../../../demo/xhr-mock';
+import notes from './notes.md';
+
+import {
+  TableAjax, TableRow, TableCell, TableHeader
+} from './table-ajax';
+import Button from '../button';
+import MultiActionButton from '../multi-action-button';
+
+const store = new Store({
+  sortOrder: 'asc',
+  sortedColumn: '',
+  currentPage: '1',
+  countryList: []
+});
+
+const handleChange = (e) => {
+  store.set({ countryList: e.data[0].items });
+};
+
+const buildRows = () => {
+  enableMock();
+
+  const rows = [
+    <TableRow key='header' as='header'>
+      <TableHeader
+        sortable name='name'
+        scope='col'
+      >
+        Country
+      </TableHeader>
+
+      <TableHeader scope='col'>Code</TableHeader>
+    </TableRow>
+  ];
+
+  store.get('countryList').map(row => (
+    rows.push(
+      <TableRow key={ row.id } uniqueID={ row.id }>
+        <TableCell>{row.name}</TableCell>
+        <TableCell>{row.value}</TableCell>
+      </TableRow>
+    )
+  ));
+
+  return rows;
+};
+
+storiesOf('Table Ajax', module).add(
+  'default',
+  () => {
+    enableMock();
+
+    const pageSize = text('pageSize', '5');
+    const paginate = boolean('paginate', false);
```

Comment:
Can we take the default value from the component? I.e. `TableAjax.defaultProps.orientation`